docs/docsite/rst/shared_snippets/galaxy_server_list.txt

```diff
@@ -1,11 +1,19 @@
 
 
+If you use the ``ansible-galaxy`` command, you need to first configure the server lists that command communicates with. You can configure this as follows:
 
-By default running ``ansible-galaxy`` will use the :ref:`galaxy_server` config value or the ``--server`` command line
-argument when it performs an action against a Galaxy server. The ``ansible-galaxy collection install`` supports
-installing collections from multiple servers as defined in the :ref:`ansible_configuration_settings_locations` file
-using the :ref:`galaxy_server_list` configuration option. To define multiple Galaxy servers you have to create the
-following entries like so:
+* Set the server list in the :ref:`galaxy_server_list` configuration option in :ref:`ansible_configuration_settings_locations` file.
+* Use the ``--server`` command line argument to limit to an individual server.
+
+To configure a Galaxy server list:
+
+#. Add the ``server_list``  option under the ``[galaxy]`` section to one or more server names.
+#. Create a new section for each server name.
+#. Set the ``url`` option for each server name.
+#. Set the ``auth_url`` option for each server name.
+#. Set the authentication for each server name. Use either the ``token`` option or the ``username`` and ``password`` options.
```

Comment:
I think `username` `password` is the older method for Automation Hub. Not sure if that ever made it into AH so maybe check with @alikins to see if we should still document it